scripts/ebpfforwindows_memory.wprp

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we merge this with the existing `wprp` file. I think we can add more than one profiles to the same file:

https://learn.microsoft.com/en-us/windows-hardware/test/wpt/3-profile-definitions#profiles